azkaban-db/build.gradle

```diff
@@ -16,6 +16,19 @@
  */
 apply plugin: 'distribution'
 
+task testJar(type: Jar, dependsOn: testClasses) {
+    from sourceSets.test.output
+    classifier = 'test'
+}
+
+configurations {
```

Comment:
Is it better to modified it to
configurations {
    testOutput.extendsFrom (testCompile)
}
as the article suggested?